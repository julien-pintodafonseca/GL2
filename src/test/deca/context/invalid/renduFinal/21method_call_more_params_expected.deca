// Description:
//      MethodCall : incompatibilité au niveau de la signature de la méthode
//      => Les paramètres passés à la fonction lors de son appel doivent correspondre à sa signature
//
// Resultats:
//      Erreur contextuelle
//      Ligne 19 : La méthode plus prend en entrée 1 paramètre au lieu de 0 paramètre

class A {
    int x;
    int plus(int y){
        y=x+y;
        return y;
    }
}

{
    A a = new A();
    println("z=", a.plus());
}
