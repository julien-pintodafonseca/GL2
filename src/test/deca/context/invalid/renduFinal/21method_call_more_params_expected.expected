src/test/deca/context/invalid/renduFinal/21method_call_more_params_expected.deca:19:18: La méthode suivante prend en entrée 1 paramètre(s) au lieu de 0 paramètre(s) : plus.
